policies/branch-policy.md

```diff
@@ -0,0 +1,100 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+## The master branch
+
+- Any type (bug fix, feature, refactoring…) of pull requests is allowed.
+
+- The development of the next minor or major release happens there.
+  Which kind of release it is is defined by OMC at some point after the
+  master branch is opened after the previous release was branched.
+  Implicitly it means that any API/ABI breaking changes are not allowed
+  until OMC defines the intended kind (minor/major) of the next release.
+
+**If the next minor branch is approved the following applies.**
```

Comment:
It's unclear to me what you mean with approval of a branch. Do you mean release?

Reply:
Basically this PR consists of two proposals for further discussion. It is still in kind-of draft state. One proposal is to have a feature branch and the other is to have a next-minor branch to develop the minor release keeping the master branch for larger refactorings/API changes.

Reply:
H